### ISSUE TITLE
[WCAG Audit] [Unique Scenario ID] [Issue Category - Product Name - Scenario]: Test

### ISSUE THREAD

**Issue by vaibhavsharma2611** [vaibhavsharma2611/vaibhavsharma2611#6]
**Important:** **Do not transfer** this issue into another repository. Instead, you can link to this issue from a project board.

## User Experience
Test

### Actual Result
Test

### Expected Result
Test

### Note
Test

### [MAS] (Microsoft Accessibility Standards) failure number and link to MAS guidance for remediation
()

### Helpful information for remediation
Link [Deque University article]() to help with remediation.

## Test Environment
- **OS:** Windows 11 22H2 (OS Build: 22621.1992)
- **Browser:** Google Chrome (Version 115.0.5790.102 (Official Build) (64-bit))
- **Assistive Technology:** NVDA (Version: 2023.1), JAWS (Version: 2023.2306.38), Developer Toolbar, aXe DevTools (Version: 4.59.2)
- **Luminance Checker Tool:** CCA - Colour Contrast Analyser (Version 3.3.0) (https://www.tpgi.com/color-contrast-checker/)
- **User Scenario Unique ID:** 
Link to the [User Scenarios Spreadsheet]()

### Pre-Requisites
Test

### Steps to Reproduce
Test

## Attachments
Test

## Acceptance Criteria
Test

**Comment by vaibhavsharma2611** [comment]
Reopeing for testing
